### DISCUSSION TITLE
change page background color

### DISCUSSION THREAD

**Discussion by kumarajay0412** [puckeditor/puck#67] [Q&A] [answered]
Suppose I have a page, I can edit its title. Is there any way I can edit and assign any custom colour to the background currently, it is white only   ?

it would be great if we can add prop  for page background color below title 

<img width="286" alt="Screenshot 2023-09-11 at 12 09 27 PM" src="https://github.com/measuredco/puck/assets/66991625/feecf7d2-0fef-4798-8166-2754285ee59f">

**Comment by chrisvxd** [comment, marked as answer]
Hey @kumarajay0412! You can add a new field to the root by adding a field to the `root` and use the `render` to render it.

If you're looking to add it the the `body`, you'll probably need to inject some styles or do some other Next.js magic.

```tsx
const config = {
  root: {
    fields: {
      title: {
        type: "text",
      },
      background: {
        type: "text",
      },
    },
    render: ({ children, title, background }) => {
      return <div style={{ background }}>{children}</div>;
    },
  },
};
```

Moving this to discussions
